apps/api/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "api",
 	"private": true,
 	"scripts": {
-		"dev": "python src/dev.py",
+		"dev": "docker compose up --no-deps --build",
```

Comment:
Issue: if the concept is to run this with turbo dev, then `docker compose up` wouldn't quite work since the containers wouldn't ever be stopped (e.g. with `docker compose down`).

Reply:
Resolved this by running a script instead:

```
docker compose up
docker compose down
```